db/schema/item.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose');
+const user = require('./user');
+
+const itemSchema = new mongoose.Schema({
+    id: String,
+    date: String,
+    tax: Number,
+    tip: Number,

```

Comment:
The item schema does not need date, tax, and tip. These fields are already in the transaction schema, which I believe should also be linked under a different field like `transaction: transaction`